smart/differential_rotation.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+from matplotlib import colors
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+from astropy.wcs import WCS
+
+from sunpy.coordinates import Helioprojective, propagate_with_solar_surface
+from sunpy.map import all_coordinates_from_map, coordinate_is_on_solar_disk, make_fitswcs_header
+
+
+def diff_rotation(im_map):
```

Comment:
is the idea here eventually to compare two magneto grams 96 mins apart? if so you can use reproject to reproject from one to the other